tests/Unit/Domain/Test_SizeOfElement.cpp

```diff
@@ -50,46 +98,33 @@ make_single_expansion_functions_of_time() noexcept {
   return functions_of_time;
 }
 
-template <bool UseMovingMesh>
 void test_1d(const std::array<double, 4>& times_to_check) noexcept {
```

Comment:
I think it would be a bit clearer to name these function `test_1d_moving_mesh` instead of only distinguishing overloads by their arguments, but I leave that up to you to decide